apps/event-service/app.py

```diff
@@ -25,7 +26,25 @@ def get_events():
     res = supabase.table("events").select(
         "name,description,start,end,id").execute()
     return res.data
-
+  
+  
+@app.get("/events/<day>")
+def get_events_by_day(day: int):
+  day_begin, day_end = None, None
+  one_day = timedelta(1)
+  
+  if day == "1":
+    day_begin = datetime(2023,2,3)
+  elif day == "2":
+    day_begin = datetime(2023,2,4)
+  elif day == "3":
+    day_begin = datetime(2023,2,5)
```

Comment:
- sure, I'll try that out.
- The weird thing about that is that it actually recognizes day as string instead of int. So, comparing using string will actually not throw an error, which I know is a bit weird